src/main/resources/db/migration/V2__enum_update.sql

```diff
@@ -0,0 +1,15 @@
+alter table orders
```

Comment:
Feel like the JPA knows how to deal with enum type in DB by `@Enumerated` annotation, do we need to make this change?

Reply:
Find a solution to solve the Mapping between Java entity and Postgres enum.
With an external library: https://github.com/vladmihalcea/hypersistence-utils
1.We need to add it as a dependency
`implementation 'io.hypersistence:hypersistence-utils-hibernate-62:3.6.0'`
2.Import it and add annotation to entity like:
```
import io.hypersistence.utils.hibernate.type.basic.PostgreSQLEnumType;
...
@Enumerated(EnumType.STRING)
@Column(name = "order_status", nullable = false, columnDefinition = "order_status")
@Type(PostgreSQLEnumType.class)
private OrderStatus orderStatus;

@Enumerated(EnumType.STRING)
@Column(name = "order_type", nullable = false, columnDefinition = "order_type")
@Type(PostgreSQLEnumType.class)
private OrderType orderType;
```

reference: 
https://vladmihalcea.com/the-best-way-to-map-an-enum-type-with-jpa-and-hibernate/
https://stackoverflow.com/questions/76567760/hibernate-6-2-mapping-between-java-enums-and-postgres-enums

Tested some APIs:
<img width="678" alt="image" src="https://github.com/OrderHere2023/OrderHere-backend/assets/117606781/04172df9-7b96-4eb0-b46b-08387a7693c2">
<img width="725" alt="image" src="https://github.com/OrderHere2023/OrderHere-backend/assets/117606781/1e2ef10c-b212-418a-89f4-f7446ddffdd4">

Reply:
👍🏻